draft-ietf-dnsop-svcb-https.md

```diff
@@ -776,21 +776,33 @@ alter their presentation or wire formats.
 The "alpn" and "no-default-alpn" SvcParamKeys together
 indicate the set of Application Layer Protocol Negotiation (ALPN)
 protocol identifiers {{!ALPN=RFC7301}}
-and associated transport protocols supported by this service endpoint.
+and associated transport protocols supported by this service endpoint (the
+"SVCB ALPN set").
 
-As with Alt-Svc {{AltSvc}}, the ALPN protocol identifier is used to
+As with Alt-Svc {{AltSvc}}, each ALPN protocol identifier is used to
 identify the application protocol and associated suite
 of protocols supported by the endpoint (the "protocol suite").
+The presence of an ALPN protocol in the SVCB ALPN set indicates that this
+service endpoint, described by TargetName and the other parameters (e.g.
+"port") offers service with the protocol suite associated with this ALPN
+protocol.
```

Comment:
```suggestion
"port") offers service with the protocol suite associated with this ALPN identifier.
```
In both cases, there are no ALPN protocols, just identifiers that reference protocols.